src/home_robot/home_robot/motion/stretch.py

```diff
@@ -602,19 +602,30 @@ def _from_manip_format(self, q_raw, q_init):
         return q
 
     def _pinocchio_pose_to_ros(self, joint_angles):
-        raise NotImplementedError
+        q = np.zeros(11)
+        q[HelloStretchIdx.BASE_X] = joint_angles[0]
+        q[HelloStretchIdx.LIFT] = joint_angles[1]
+        q[HelloStretchIdx.ARM] = (
+            joint_angles[2] + joint_angles[3] + joint_angles[4] + joint_angles[5]
+        )
```

Comment:
don't we already have a manip ik pose function that should do this?
if it's pin specific it should probably be in that IK solver?